SpatialGDK/Source/SpatialGDK/Public/Interop/RPCs/CrossServerRPCService.h

```diff
@@ -75,6 +78,7 @@ class SPATIALGDK_API CrossServerRPCService
 	ActorCanExtractRPCDelegate CanExtractRPCDelegate;
 	ExtractRPCDelegate ExtractRPCCallback;
 	const FSubView* SubView;
+	const FSubView* WorkerEntitySubView;
```

Comment:
nit - having a specific name for this subview makes me want a specific name for "SubView" also...